src/test/java/sisyphus/TaskListTest.java

```diff
@@ -0,0 +1,64 @@
+package sisyphus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+
+
+public class TaskListTest {
+
+    @Test
+    void newListIsEmpty() {
+        TaskList list = new TaskList(new ArrayList<>());
+        assertTrue(list.isEmpty());
+        assertEquals(0, list.size());
+    }
+
+    @Test
+    void addIncreasesSizeAndGetReturnsTask() {
```

Comment:
good job on comprehensive unit tests, having full code coverage of all logical paths!